.pre-commit-config.yaml

```diff
@@ -10,24 +10,30 @@ repos:
       - id: check-merge-conflict
       - id: check-symlinks
       - id: check-toml
+        exclude: llama_index/_static
       - id: check-yaml
+        exclude: llama_index/_static
       - id: detect-private-key
       - id: end-of-file-fixer
+        exclude: llama_index/_static
       - id: mixed-line-ending
+        exclude: llama_index/_static
       - id: trailing-whitespace
+        exclude: llama_index/_static
   - repo: https://github.com/charliermarsh/ruff-pre-commit
     rev: v0.1.5
 
     hooks:
       - id: ruff
         args: [--fix, --exit-non-zero-on-fix]
+        exclude: llama_index/_static
   - repo: https://github.com/psf/black-pre-commit-mirror
     rev: 23.10.1
     hooks:
       - id: black-jupyter
         name: black-src
         alias: black
-        exclude: docs/
+        exclude: ^(docs/|llama_index/_static)
```

Comment:
is this only excluding `docs/|llama_index/_static)`

Reply:
it's a regex, exclude `docs/` or `llama_index/_static` from being run with `black`